src/linear_algebra/exterior_algebra.lean

```diff
@@ -0,0 +1,152 @@
+/-
+Copyright (c) 2020 Adam Topaz. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Zhangir Azerbayev, Adam Topaz, Eric Wieser.
+-/
+
+import algebra.ring_quot
+import .tensor_algebra
```

Comment:
Please don't use relative imports in mathlib.

Reply:
Fixed